### PR TITLE
Ultimate Christmas

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About The Pull Request

My late christmas gift.
All sprites are by `thatredbastard`

- Adds buff Rudolta as a joke ALEPH;
- Resprites Rudolta of the Sleigh.

## Why It's Good For The Game

- A silly meme that will probably be normally unavailable after a while;
- A better sprite.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/ego_gifts.dm
Comment (LanceSmites328) [inline code/modules/mob/living/carbon/human/ego_gifts.dm:1344]: The single most balanced gift.

**Comment by Kitsunemitsu** [comment]
Can you make it enable on the christmas event automatically

**Review by LanceSmites328 — APPROVED** [review]
The fact I can't approve this more than once at a time is a crying shame.

**Pull request merged**
